drivers/storage/portworx/component/telemetry.go

```diff
@@ -23,10 +26,27 @@ const (
 	TelemetryPropertiesFilename = "ccm.properties"
 	// TelemetryArcusLocationFilename is name of the file storing the location of arcus endpoint to use
 	TelemetryArcusLocationFilename = "location"
+	// TelemetryCertName is name of the telemetry cert.
```

Comment:
I thought about this, but feel it is part of telemetry, hence I have separated the API to deployMetricsCollector

Reply:
Technically it is not part of telemetry from our side, although it could be seen as one from customer standpoint. 
From our side telemetry/ccm-service is meant to only collect diags from portworx while metrics collector is only sending back metrics to Pure1.
Anyway, it is probably a minor thing right now as these are not big components. I will leave it up to you.